src/gui/guiEngine.cpp

```diff
@@ -75,9 +75,8 @@ video::ITexture *MenuTextureSource::getTexture(const std::string &name, u32 *id)
 	if (name.empty())
 		return NULL;
 
-	m_to_delete.insert(name);
-
 #if ENABLE_GLES
+	m_to_delete.insert(name);
```

Comment:
This should be moved next to the `addTexture` call it belongs to

Reply:
Done